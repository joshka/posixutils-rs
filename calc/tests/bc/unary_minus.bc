-0
-1
a = 20
-a
a = -20
-a
- - 2.45
quit
