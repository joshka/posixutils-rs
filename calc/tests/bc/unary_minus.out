0
-1
-20
20
2.45
